ads/google/test/test-doubleclick.js

```diff
@@ -0,0 +1,363 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  createFixtureIframeFromHtml,
+  pollForLayout,
+  poll,
+} from '../../../testing/iframe';
+
+const waitTime = 20 * 1000;
+
+const doubleClickTest = function(config) {
```

Comment:
Removing the function entirely would require that I duplicate most of it 7 times, as all the tests have virtually the same skeletal structure, with only expected values changed here and there.